cmd/beaker/main.go

```diff
@@ -209,3 +211,25 @@ func login() error {
 	}
 	return config.WriteConfig(beakerConfig, config.GetFilePath())
 }
+
+// confirm prompts the user for a yes/no answer and defaults to no.
+// Returns true, nil if the user enters yes.
+func confirm() (bool, error) {
+	reader := bufio.NewReader(os.Stdin)
+	for {
+		input, err := reader.ReadString('\n')
+		if err != nil {
+			return false, err
+		}
+		input = strings.TrimSuffix(input, "\n")
```

Comment:
TrimSpace is nicer. A leading or trailing space is common.
```suggestion
		input = strings.TrimSpace(input)
```